verifier.sh

```diff
@@ -304,19 +305,21 @@ else
 fi
 fab --show=everything test
 
-cd openquakeplatform/test
+# download and install vulnerability development data
+wget http://ftp.openquake.org/oq-platform/vulnerability/dev-data.json.bz2
+python ./manage.py loaddata dev-data.json.bz2
+
 export PYTHONPATH=\$(pwd)
-cp config.py.tmpl config.py
+cp openquakeplatform/test/config.py.tmpl openquakeplatform/test/config.py
 export DISPLAY=:1
-./test_isc.py
-
+python /usr/bin/nosetests -v --with-xunit --xunit-file=xunit-platform-dev.xml  openquakeplatform/test
 sleep 3
-cd ~/$GEM_GIT_PACKAGE
-. platform-env/bin/activate
-cd openquakeplatform
 fab stop
 "
 
+    scp "${lxc_ip}:$GEM_GIT_PACKAGE/openquakeplatform/xunit-platform-dev.xml" "out/" || true
+    scp "${lxc_ip}:$GEM_GIT_PACKAGE/openquakeplatform/latest_page.png" "out/latest_page_dev.png" || true
```

Comment:
Reminder: you'll need to update also the Jenkins configuration to move this output into the artifacts